repos/system_upgrade/common/models/grubcfg.py

```diff
@@ -0,0 +1,8 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemFactsTopic
+
+
+class GrubCfg(Model):
+    topic = SystemFactsTopic
+
+    grub_cfg = fields.String()
```

Comment:
Similar to models defined above. In this case the content of the file is not even parsed, which is not good. We should introduce model that reflects the configuration. Let's look at what we could deliver. In case it's not a big deal, I think we should give it some love.

Reply:
let's just produce info / facts we use instead of content of the file